### DISCUSSION TITLE
the latest Glint360k

### DISCUSSION THREAD

**Discussion by nadongjin** [leondgarse/Keras_insightface#62] [Q&A]
Is there a way to download the latest Glint360k?
It's so hard because I don't know Chinese.
Can you tell me how to download this?
Thank you...

**Comment by leondgarse** [comment]
Have you tried their magnet uri: `magnet:?xt=urn:btih:E5F46EE502B9E76DA8CC3A0E4F7C17E4000C7B1E&dn=glint360k`? It's so large that no where I can share you mine.

**Reply by nadongjin** [reply]
Thank you.
I'm downloading it through Baidu right now.
It's very slow.

**Reply by leondgarse** [reply]
Uh, ya, Baidu is limited at about `100k/s` without paying them, no matter where. That will take weeks...
If you can make Baidu work, try this one from aliyun drive: `https://www.aliyundrive.com/s/NKeQ3FvbPjn`. It's downloading / uploading speed is not limited. That's the `train.idx` from `Glint360k` dataset. If your downloading works, I will try upload the 130G `train.rec` there.

**Reply by nadongjin** [reply]
I can't sign up for aliundrive membership.
For Korean mobile numbers, the authentication code will not be sent?
i have succeeded in signing up for Alibaba Cloud, but i cannot log in to Aliyundrive.

**Reply by nadongjin** [reply]
I could download it here.
Anyway, I am grateful to you.
https://github.com/tamerthamoqa/facenet-pytorch-glint360k#training-and-testing-datasets
#glint360k dataset (unpacked and available on my Google Drive): Drive

However, the glint360k dataset in baidu has been updated as of December 9.
It's too slow, but I think it's better to get this.

**Reply by leondgarse** [reply]
- I don't know how Aliyun drive code works either.
- Such a large space on Google drive! only 15G for me... I haven't download that December one either. Actually, seldom do I run training on Glint360k,  cause it's too large. May only try when my recipe on MS1MV3 seems all good.
